### PR TITLE
Adressing review suggestions for Version 1.0.0

### PR DESCRIPTION
**Pull request by Felix-Kummer**
Adding changes requested in review [https://github.com/nf-core/rangeland/pull/8](https://github.com/nf-core/rangeland/pull/8) for the first release of the Rangeland pipeline in nf-core [https://github.com/nf-core/rangeland/pull/7](https://github.com/nf-core/rangeland/pull/7) .

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,5 @@ testing/
 testing*
 *.pyc
 null/
+.nf-test.log
+.nf-test/*
```
Comment (mashehu) [inline .gitignore:11]:
```suggestion
.nf-test*
```

File: .nf-core.yml [hunk 2]
```diff
@@ -2,6 +2,7 @@ bump_version: null
 lint:
   files_exist:
     - conf/igenomes.config
+    - conf/igenomes_ignored.config
```
Comment (mashehu) [inline .nf-core.yml:5]: is this correct, @mirpedrol, I thought we needed this even when not using igenomes

File: docs/usage.md [hunk 3]
```diff
@@ -277,33 +321,52 @@ The time series stack output can be enabled using:
...
+By default, both visualization methods are enabled.
+Note that the mosaic visualization is required to be enabled when using the `test` and `test_full` profiles to allow the pipeline to check the correctness of its results.
 
 The visualizations can be enabled using:
 
 ```bash
-mosaic_visualization  = '[boolean]'
-pyramid_visualization = '[boolean]'
+--mosaic_visualization  = '[boolean]'
+--pyramid_visualization = '[boolean]'
```
Comment (mashehu) [inline docs/usage.md:335]:
```suggestion
mosaic_visualization  = 'true'
pyramid_visualization = 'true'
```
maybe also add a `title=paramters.yml` to the code block and switch the language. for reproducibility I would discourage cli paramter input and nudge people to use a paramter file instead.

File: docs/usage.md [hunk 4]
```diff
@@ -277,33 +321,52 @@ The time series stack output can be enabled using:
...
+However, analysis-ready-data (aka. ARD or level-2 data) and the results of time series analyses (aka. level-3 data) maybe be interesting to certain users.
+The files for preprocessing and higher level processing can, therefore, be published by setting the `--save_ard` and `--save_tsa` to `true`, respectively.
 
-The number of threads can be passed using:
+Publishing of intermediate data can be enabled using:
 
 ```bash
---force_threads '[integer]'
+--save_ard  = '[boolean]'
+--save_tsa  = '[boolean]'
```
Comment (mashehu) [inline docs/usage.md:348]: same as above

File: docs/usage.md [hunk 5]
```diff
@@ -277,33 +321,52 @@ The time series stack output can be enabled using:
...
 ```
 
-The default value is 2.
+### Module output publishing
+
+Certain users may not be interested in the pipeline's outputs.
```
Comment (mashehu) [inline docs/usage.md:353]: why would one want to run a pipeline, while not being interested in the outputs? 🤔

File: docs/usage.md [hunk 6]
```diff
@@ -348,17 +412,22 @@ You can also generate such `YAML`/`JSON` files via [nf-core/launch](https://nf-c
...
 
 ### Reproducibility
 
-It is a good idea to specify a pipeline version when running the pipeline on your data. This ensures that a specific version of the pipeline code and software are used when you run your pipeline. If you keep using the same tag, you'll be running the same version of the pipeline, even if there have been changes to the code since.
+It is a good idea to specify a pipeline version when running the pipeline on your data.
+This ensures that a specific version of the pipeline code and software are used when you run your pipeline.
+If you keep using the same tag, you'll be running the same version of the pipeline, even if there have been changes to the code since.
 
-First, go to the [nf-core/rangeland releases page](https://github.com/nf-core/rangeland/releases) and find the latest pipeline version - numeric only (eg. `1.3.1`). Then specify this when running the pipeline with `-r` (one hyphen) - eg. `-r 1.3.1`. Of course, you can switch to another version by changing the number after the `-r` flag.
+First, go to the [nf-core/rangeland releases page](https://github.com/nf-core/rangeland/releases) and find the latest pipeline version - numeric only (eg. `1.3.1`).
```
Comment (mashehu) [inline docs/usage.md:429]:
```suggestion
First, go to the [nf-core/rangeland releases page](https://github.com/nf-core/rangeland/releases) and find the latest pipeline version - numeric only (eg. `1.0.0`).
```
I would provide an example that actually works for users.

**Review by mashehu — APPROVED** [review]

File: .nf-core.yml (hunk 2 above)
Comment (mirpedrol) [reply .nf-core.yml:5]: It is correct, this file is to allow the use of `params.igenomes_ignore`, but if the igenomes is completely excluded from the pipeline, then we don't need any of these files.

Comment (mashehu) [reply .nf-core.yml:5]: okay, thanks!

File: docs/usage.md (hunk 3 above)
Comment (Felix-Kummer) [reply docs/usage.md:335]: I would need to change that for every parameter in usage.md 
Previously I was told to use cli parameter input.
Is there a guideline for that?

File: docs/usage.md (hunk 5 above)
Comment (Felix-Kummer) [reply docs/usage.md:353]: This was requested by a colleague for evaluating workflow execution approaches using different task placement strategies.

**Pull request merged**